Dev Merge - Audit Report & Content Fixes (#62)
babylonlabs-io/babylonlabs.github.io: docs/guides/security/audit_reports.mdx

sidebar_position: 1
---

import BrowserOnly from '@docusaurus/BrowserOnly';

# Audit Reports

> Babylon protocol is secure and audited by industry leading web3 security firms. 
> Since the launch of the protocol is phased, specific audit reports are conducted. 

Reviewer: ```suggestion
Babylon protocol is audited by industry leading web3 security firms.
Below you can find the audits conducted for the code base associated with
the different phases of the launch.
```

### Phase 1 Audits

Audits for the Phase 1 of the Babylon Bitcoin Staking Protocol are conducted by three different firms.

<CardSection id="Babylon Security Audit Reports">
  <BrowserOnly>
    {() => (
      <Card

>         title="Coinspect Audit Report"

Reviewer: ```suggestion
        title="Coinspect Phase-1 Audit Report"
```

        to={new URL('/assets/files/coinspect-phase1-audit.pdf', window.location.origin).href}

>         description="Source Code Audit - Phase 1 (June 2024)"

Reviewer: ```suggestion
        description="Coinspect Source Code Audit - Phase 1 (June 2024)"
```

        target="_blank"
        rel="noopener noreferrer"
      />
    )}
  </BrowserOnly>
  <BrowserOnly>
    {() => (
      <Card

>         title="Coinspect Audit Report Incremental "

Reviewer: ```suggestion
        title="Coinspect Phase-1 Incremental Audit Report"
```

        to={new URL('/assets/files/coinspect-phase1-audit-incremental.pdf', window.location.origin).href}

>         description="Source Code Audit - Phase 1 Cap3 (October 2024)"

Reviewer: ```suggestion
        description="Zellic Source Code Audit - Phase 1 Cap-3 updates (October 2024)"
```

        target="_blank"
        rel="noopener noreferrer"
      />
    )}
  </BrowserOnly>
  <BrowserOnly>
    {() => (
      <Card
        title="Zellic"
        to={new URL('/assets/files/zellic-phase1-audit.pdf', window.location.origin).href}

>         description="Blockchain Security Assessment (June 2024)"

Reviewer: ```suggestion
        description="Zellic Source Code Audit - Phase 1 (June 2024)"
```

        target="_blank"
        rel="noopener noreferrer"
      />
    )}
  </BrowserOnly>
  <BrowserOnly>
    {() => (
      <Card
        title="Cantina"
        to={new URL('/assets/files/cantina-phase1-competition.pdf', window.location.origin).href}

>         description="Bitcoin Staking Scripts Competition (January 2024)"

Reviewer: ```suggestion
        description="Sherlock Source Code Security Competition - Phase 1 (June 2024)"
```

        target="_blank"
        rel="noopener noreferrer"
      />
    )}
  </BrowserOnly>
</CardSection>

### Phase 2 Audits

Audits for the Babylon Genesis Chain are conducted by [Coinspect](https://www.coinspect.io/) and [Zellic](https://zellic.io/).

<CardSection id="Babylon Security Audit Reports">
  <BrowserOnly>
    {() => (
      <Card
        title="Coinspect Phase 2 Audit Report"
        to={new URL('/assets/files/coinspect_babylon_phase_2_audit_2025_03.pdf', window.location.origin).href}

>         description="Babylon Genesis Chain Audit (March 2025)"

Reviewer: ```suggestion
        description="Coinspect Babylon Genesis Chain Audit (March 2025)"
```

        target="_blank"
        rel="noopener noreferrer"
      />
    )}
  </BrowserOnly>
  <BrowserOnly>
    {() => (
      <Card
        title="Zellic Babylon Genesis Audit Report"
        to={new URL('/assets/files/zellic_babylon_genesis_chain_audit_2025_03.pdf', window.location.origin).href}

>         description="Babylon Genesis Chain Audit (March 2025)"

Reviewer: ```suggestion
        description="Zellic Babylon Genesis Chain Audit (March 2025)"
```
